docs/configuration/elements.md

```diff
@@ -67,7 +67,7 @@ Namespace: `options.elements.line`, global line options: `Chart.defaults.element
 | `borderDashOffset` | `number` | `0.0` | Line dash offset. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/lineDashOffset).
 | `borderJoinStyle` | `string` | `'miter'` | Line join style. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/lineJoin).
 | `capBezierPoints` | `boolean` | `true` | `true` to keep Bézier control inside the chart, `false` for no restriction.
-| `cubicInterpolationMode` | `string` | `'default'` |  Interpolation mode to apply. [See more...](/charts/line.md/#cubicinterpolationmode)
+| `cubicInterpolationMode` | `string` | `'default'` |  Interpolation mode to apply. [See more...](/charts/line.html#cubicinterpolationmode)
```

Comment:
I'm not sure that the extension change is required. I had a look at some of the other links in the docs and they usually use `file.md#anchor` so removing the `/` should be enough

Reply:
Just checked, unfortunately not enough.

Reply:
https://www.chartjs.org/docs/latest/charts/line.md#cubicinterpolationmode
doesn't work

Reply:
🤔 are you testing with the `npm run docs:dev` command? That should build everything the way the CI does but give you a dev server on localhost:8080

Reply:
Oh, I just checked in the browser, so you're probably right enough to remove `/`. Could you to close this PR?

Reply:
if you want to push an update to just remove the slash, I can merge this 😄

Reply:
Done, please check is it right :)

Reply:
I tested locally and confirmed that it works 😃

Reply:
Great! Thanks for good mood ;)